src/frontend/src/test-e2e/views.ts

```diff
@@ -32,16 +32,6 @@ export class WelcomeView extends View {
     await this.browser.$("#loginButton").click();
     await this.browser.$("#addNewDeviceButton").click();
   }
-
-  async recover(): Promise<void> {
```

Comment:
Why this move? Does it have something to do with clearing the local storage on logging out?

Reply:
Yes, see explanation [here](https://github.com/dfinity/internet-identity/pull/1929#issuecomment-1734928421).